front/src/config/i18n/fr.json

```diff
@@ -917,7 +919,11 @@
           "friday": "Vendredi",
           "saturday": "Samedi",
           "sunday": "Dimanche"
-        }
+        },
+        "house": "Maison"
+      },
+      "warning": {
+        "houseWithoutCoordinate": "Attention !! Votre maison n'a pas de coordonnées renseignées. Ce déclencheur ne peut fonctionner sans cette donnée. Veuillez-vous rendre dans Paramètes/Maisons et cliquer sur la carte pour en associer une."
```

Comment:
Could you be less "stressed" in the message? The "Attention !!" is not necessary.